lib/rock/webapp/tasks/api.rb

```diff
@@ -7,6 +8,9 @@ module Tasks
             class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
+                
+                logger.formatter = GrapeLogging::Formatters::Json.new
+                use GrapeLogging::Middleware::RequestLogger, {logger: logger}
```

Comment:
No, tried it first.
The documentation says "on top of your API file":
https://github.com/aserafin/grape_logging